Apps/VisionOS/VisionOS/AppDelegate.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
+
+import CioDataPipelines
+
+class AppDelegate: NSObject, UIApplicationDelegate {
+    func application(
+        _ application: UIApplication,
+        didFinishLaunchingWithOptions launchOptions: [UIApplication
+            .LaunchOptionsKey: Any]? = nil
+    ) -> Bool {
+        /*
+
+         You will need cdpApiKey to initialize CustomerIO
+         Learn more about it here: https://customer.io/docs/sdk/ios/quick-start-guide/#prerequisites
+         */
+
+        let workspaceSettings = AppState.shared.workspaceSettings
+        if workspaceSettings.isSet() {
+            CustomerIO.initialize(
+                withConfig:
+                SDKConfigBuilder(cdpApiKey: workspaceSettings.cdpApiKy)
```

Comment:
Is `cdpApiKy` a legit property name of `workspaceSettings` or a mistake ?

Reply:
Good call out. It was more valid when it was siteId and ApiKey. I can rename this one to be more like "SourceSetting" or something similar

Reply:
You got it in the next comment ;)